md/uid-modal-tutorial.md

```diff
@@ -1,5 +1,9 @@
 # How to create a modal window using CSS
 
+::: warning
+**:fa-exclamation-triangle: Warning:** This tutorial is deprecated after 7.8 because of the introduction of the modal container. Please read more about the modal on the widgets page : [Modal Container](widgets.md)
```

Comment:
```suggestion
**:fa-exclamation-triangle: Warning:** This tutorial is deprecated after 7.8 because of the introduction of the modal container. For more information, read the [modal container](widgets.md) page.
```

Reply:
You need to place an anchor in the widget page and link this warning directly to the anchor.